src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -10,6 +10,7 @@
     <ProjectReference Include="..\Microsoft.ML.CpuMath\Microsoft.ML.CpuMath.csproj" />
     <ProjectReference Include="..\Microsoft.ML.LightGbm\Microsoft.ML.LightGbm.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Mkl.Components\Microsoft.ML.Mkl.Components.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.Recommender\Microsoft.ML.Recommender.csproj" />
```

Comment:
You will also need to update the AutoML nupkgproj as well:


[https://github.com/dotnet/machinelearning/blob/461b7d38c06d64e53c76e29770665c4b0bb44e72/pkg/Microsoft.ML.AutoML/Microsoft.ML.AutoML.nupkgproj#L9-L11](https://github.com/dotnet/machinelearning/blob/461b7d38c06d64e53c76e29770665c4b0bb44e72/pkg/Microsoft.ML.AutoML/Microsoft.ML.AutoML.nupkgproj#L9-L11)